Sources/SwiftInspectorVisitors/TypeSyntaxVisitor.swift

```diff
@@ -267,7 +271,7 @@ extension TypeProperties.MergeError {
 }
 
 extension TypeProperties {
-  func merge(with other: TypeProperties?) throws -> TypeProperties {
+  public func merge(with other: TypeProperties?) throws -> TypeProperties {
```

Comment:
Note to self while I review: this extension method is used by the analyzer, so it should also be `public`.